src/wp-admin/js/edit-comments.js

```diff
@@ -631,6 +831,15 @@ commentReply = {
 		return false;
 	},
 
+	/**
+	 * Close the comment edit or reply form.
+	 *
+	 * @since 3.5.0
+	 *
+	 * @memberOf commentReply
+	 *
+	 * @returns {void}
+	 */
```

Comment:
Please start inline comments in this function with an uppercase letter and end them with full stops

Reply:
Could you also change `replyrow is not showing?` so it's not a question